debian/salsa-ci.yml

```diff
@@ -383,41 +355,8 @@ build mariadbclient consumer Python-MySQLdb:
     variables:
       - $CI_COMMIT_TAG != null && $SALSA_CI_ENABLE_PIPELINE_ON_TAGS !~ /^(1|yes|true)$/
 
-libmysql* to libmariadb* upgrade:
```

Comment:
Is this not needed to be tested anymore?

Reply:
Since Debian Sid now has MariaDB 10.6, we can't do any upgrade tests in
Debian Sid for the 10.5 branch anymore. It would just fail with downgrade
errors.

Reply:
MySQL in Debian exists only in Sid, we can't use other Debian releases for these tests either.